build.gradle

```diff
@@ -2333,6 +2333,8 @@ project(':streams:streams-scala') {
 
     testImplementation libs.junitJupiter
     testImplementation libs.easymock
+    testImplementation libs.mockitoCore
```

Comment:
Wait...why do we need to add a dependency for this? I thought the whole point of the change was to align this test with others in the same file -- shouldn't that mean whatever dependencies are needed are already there?

Reply:
I checked. looks like no tests in the streams scala tests used mockito yet 🤷